single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java

```diff
@@ -4,26 +4,44 @@
 
 public class ServerConfiguration {
 
+    private static final int DEFAULT_PORT = 443;
+    private static final String DEFAULT_HEALTH_PATH = "";
     private static final long DEFAULT_DNS_LOOKUP_REFRESH_PERIOD_IN_SECONDS = TimeUnit.MINUTES.toSeconds(5);
     private static final long DEFAULT_CONNECTION_HEALTH_CHECK_PERIOD_IN_SECONDS = 30;
+    private static final long DEFAULT_READ_TIMEOUT_IN_SECONDS = -1; // It means there is no read timeout
 
     private final String hostname;
     private final int port;
     private final String healthPath;
     private final long connectionHealthCheckPeriodInSeconds;
     private final long dnsLookupRefreshPeriodInSeconds;
-
+    private final long readTimeoutInSeconds;
```

Comment:
I would use a `Duration` so that it's possible to have sub-second timeout

Reply:
Well, yes, but we have other periods as `long` values. I would stick to this single style, but not principal.